taxonomies/categories.txt

```diff
@@ -39463,12 +39477,14 @@ ciqual_food_name:en:Blueberry, raw
 ciqual_food_name:fr:Myrtille, crue
 
 <en:Blueberries
+<en:Fresh fruits
+en:Fresh blueberries
 fr:Myrtilles fraîches
 season_in_country_fr:en: 7,8,9
 
 <en:Blueberries
-en:Frozen raw Blueberry
-fr:Myrtille surgelée crue
+en:Frozen raw bluberries, Frozen raw blueberry
```

Comment:
I'm wondering if we should put "Frozen blueberries" as the main name for "Frozen raw blueberries".  (also there'a missing in "bluberries")

Reply:
ok just saw below that we have another "Frozen blueberries" category. I think we should merge them.